hack/Vagrantfile.freebsd

```diff
@@ -11,7 +11,9 @@ Vagrant.configure("2") do |config|
     sh.inline = <<~SHELL
       #!/usr/bin/env bash
       set -eux -o pipefail
+      freebsd-version -kru
       kldload nullfs
+      sed -i '' 's/latest/release_2/' /usr/local/etc/pkg/repos/FreeBSD.conf
```

Comment:
Could you add a comment line to explain the context, then LGTM

Reply:
Done thanks